css/styles.css

```diff
@@ -0,0 +1,131 @@
+.gray-background {
+  background-color: #282828;
+  font-family: sans-serif;
+}
+
+/* nav bar styles */
+.nav-bar {
+  justify-content: space-between;
+  background-color: #e6f2fc;
+  width: 90%;
+  height: 45px;
+  align-items: center;
+}
+
+.left-div-nav {
+  display: flex;
+  justify-content: space-between;
+  flex-basis: 250px;
+}
+
+.home-link {
+  text-decoration: none;
+  color: black;
+  font-size: larger;
+  font-weight: bold;
+  padding-left: 10px;
+}
+
+.store-info-link {
+  text-decoration: none;
+  color: #5d9af7;
+  font-size: larger;
+  font-weight: bold;
+}
+
+.search-button {
+  border: solid #4582f7;
+  width: 26%;
+  height: 30px;
+  margin-right: 10px;
+  border-radius: 5px;
```

Comment:
For small stuff. use `rem` instead of `px`: https://lfz-lms.herokuapp.com/code-guides/Learning-Fuze/curriculum/effective-css